lib/sass/calcite-web/icons/_calcite-ui.scss

```diff
@@ -208,6 +208,16 @@
   }
 }
 
+@mixin svg-icon() {
+  fill: currentColor;
+  pointer-events: none;
```

Comment:
lets click pass through the `path` tag to get picked up by whatever dom node the icon is within